docs/11_kripke/kripke.rst

```diff
@@ -9,6 +9,19 @@ Kripke source code is near-final at this point. The problem to run is yet to be
 Purpose
 =======
 
+Kripke is a simple, scalable, 3D Sn deterministic particle transport code.
+Its primary purpose is to research how data layout, programming paradigms and architectures affect the implementation and performance of Sn transport.
+A main goal of Kripke is to investigate how different data-layouts affect instruction, thread, and task level parallelism, and the implications on overall solver performance.
+
+Kripke supports storage of angular fluxes (Psi) using all six striding orders (or "nestings") of Directions (D), Groups (G), and Zones (Z), and provides computational kernels specifically written for each of these nestings.
+Most Sn transport codes are designed around one of these nestings, which is an inflexibility that leads to software engineering compromises when porting to new architectures and programming paradigms.
+
+Early research has found that the problem dimensions (zones, groups, directions, scattering order) and the scaling (number of threads and MPI tasks), can make a profound difference in the performance of each of these nestings.
+To our knowledge, this is a capability unique to Kripke, and should provide key insight into how data-layout affects Sn solver performance.
```

Comment:
@rchen20 We need details in the code modification section on what may be changed, and what should stay the same.  If I am not mistaken, the data ordering (GDZ) needs to stay the same,. while the execution policy can be changed and optimizations like loop tiling are allowed.